osu.Game.Rulesets.Catch/CatchSkinComponentLookup.cs

```diff
@@ -12,8 +12,6 @@ public CatchSkinComponentLookup(CatchSkinComponents component)
         {
         }
 
-        protected override string RulesetPrefix => "catch"; // todo: use CatchRuleset.SHORT_NAME;
-
-        protected override string ComponentName => Component.ToString().ToLowerInvariant();
```

Comment:
In all of these `GameplaySkinComponentLookup` classes, these properties are unused - they were likely from a different time.